services/honeycomb-agent/shipit.yaml

```diff
@@ -1,13 +1,16 @@
-# honeycomb-agent is deployed via helm. I don't know a great deal about it, but it seems we can update the deployment with
+# honeycomb-agent is deployed via helm. I don't know a great deal about it, but it
+# seems we can update the deployment with
+#
```

Comment:
Slightly confusing with the newline - I assume the 'with' is supposed to continue with the `help upgrade ...` bit?

Reply:
Will handle in followup.